04-loop.md

```diff
@@ -277,7 +279,9 @@ Since she's still learning how to use the shell,
 she decides to build up the required commands in stages.
 Her first step is to make sure that she can select the right files --- remember,
 these are ones whose names end in 'A' or 'B', rather than 'Z'. Starting from her home directory, Nelle types:
-
+A graphical representation of how the loop will be processed is shown below:
+{insert image here: https://www.dropbox.com/s/vpn7k4r1gxiaaxo/Shell%20Script%20For%20Loop%20Flow%20Chart.jpg?dl=0]
+Figure: Flowcart of For Loop in Shell
```

Comment:
Thanks Ashwin. Deleted lines 282-284.

Cheers,

Dr Francis Gacenga
QCIF eResearch Analyst
Office of Research Development | Research and Innovation Division | University of Southern
Queensland | Toowoomba, Queensland | 4350| Australia
Phone: +61 7 4687 5640 | 0477 072 707 | Email: gacenga@usq.edu.au<mailto:gacenga@usq.edu.au> | Twitter: https://twitter.com/USQ_eResearch
Website: https://www.usq.edu.au/eresearch |QRIScloud: https://www.qriscloud.org.au/

From: Ashwin Srinath [mailto:notifications@github.com]
Sent: Monday, 23 May 2016 8:47 PM
To: swcarpentry/shell-novice <shell-novice@noreply.github.com>
Cc: Francis Gacenga <Gacenga@usq.edu.au>; Mention <mention@noreply.github.com>
Subject: Re: [swcarpentry/shell-novice] Update 04-Loop.md (#366)


In 04-loop.md<https://github.com/swcarpentry/shell-novice/pull/366#discussion_r64202319>:

> @@ -277,7 +279,9 @@ Since she's still learning how to use the shell,

>  she decides to build up the required commands in stages.

>  Her first step is to make sure that she can select the right files --- remember,

>  these are ones whose names end in 'A' or 'B', rather than 'Z'. Starting from her home directory, Nelle types:

> -

> +A graphical representation of how the loop will be processed is shown below:

> +{insert image here: https://www.dropbox.com/s/vpn7k4r1gxiaaxo/Shell%20Script%20For%20Loop%20Flow%20Chart.jpg?dl=0]

> +Figure: Flowcart of For Loop in Shell

You are linking to the image already above (line 231). You can get rid of lines 282-284

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/swcarpentry/shell-novice/pull/366/files/517d6fd3fe0e4ed43e7eb7ec3d00969ec6922006#r64202319>


_____________________________________________________________
This email (including any attached files) is confidential and is for the intended recipient(s) only. If you received this email by mistake, please, as a courtesy, tell the sender, then delete this email.

The views and opinions are the originator's and do not necessarily reflect those of the University of Southern Queensland. Although all reasonable precautions were taken to ensure that this email contained no viruses at the time it was sent we accept no liability for any losses arising from its receipt.

The University of Southern Queensland is a registered provider of education with the Australian Government.
(CRICOS Institution Code QLD 00244B / NSW 02225M, TEQSA PRV12081 )